app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="navigation">
+  <ul class="flex justify-between blue-btn items-center">
+    <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), category_path(params[:category_id]), class: 'nav-link' %></li>
+    <li><h2>Add a new Purchase</h2></li>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: 'nav-link' %></li>
```

Comment:
#### New transaction form

As per project requirements, there should be a `mandatory` drop down button that have a list of all created categories to choose from. The transaction that is saved should be for the category that is picked on the drop down list of available categories.

![image](https://github.com/iamsjunaid/budget-bee/assets/38088818/e134fce9-fc52-4c98-b267-20eb748c4f38)

![image](https://github.com/iamsjunaid/budget-bee/assets/38088818/528d71fb-ce4a-4f69-ad32-2e69a06fafa5)